test/quantization/test_quantize_fx.py

```diff
@@ -1592,6 +1592,57 @@ def forward(self, x):
         assert hasattr(m, "mods1_1_packed_weight_0")
         assert hasattr(m, "mods2_packed_weight_0")
 
+    def test_state_dict(self):
+        """ Make sure packed params appear in state_dict
+        """
+
+        # test linear packed weight
+        class M1(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.w = torch.rand(4, 30)
+                self.b = torch.rand(4)
+
+            def forward(self, x):
+                return F.linear(x, self.w, self.b)
+
+        m = M1().eval()
+        qconfig_dict = {"": default_qconfig}
+        m = prepare_fx(m, qconfig_dict)
+        m = convert_fx(m)
+        state_dict = m.state_dict()
+        self.assertTrue("_packed_weight_0" in state_dict)
+
+        # test conv packed weight
+        class M2(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.w = torch.rand(3, 3, 3, 3)
+                self.b = torch.rand(3)
+                self.stride = (1, 1)
+                self.padding = (0, 0)
+                self.dilation = (1, 1)
+                self.groups = 1
+
+            def forward(self, x):
+                return F.conv2d(x, self.w, self.b, self.stride, self.padding, self.dilation, self.groups)
+
+        m = M2().eval()
+        qconfig_dict = {"": default_qconfig}
+        m = prepare_fx(m, qconfig_dict)
+        m = convert_fx(m)
+        state_dict = m.state_dict()
+        self.assertTrue("_packed_weight_0" in state_dict)
+
+        # test load
+        ref_weight, ref_bias = torch.ops.quantized.conv2d_unpack(state_dict["_packed_weight_0"])
+        m = M2().eval()
+        m = prepare_fx(m, qconfig_dict)
+        m = convert_fx(m)
+        m.load_state_dict(state_dict)
```

Comment:
Can we run forward on the 2nd model with same set of inputs to compare with 1st model?
This will ensure that we loaded all quantization related params correctly from the state_dict